bin/desi_proc_night

```diff
@@ -102,6 +102,10 @@ def parse_args():
                         help="If set then the code will NOT raise an error "
                              + "if asked to link psfnight calibrations "
                              + "without fiberflatnight calibrations.")
+    parser.add_argument("--still-acquiring", action=argparse.BooleanOptionalAction,
+                        help="in --daily mode, assume more data is still coming even if "
```

Comment:
Please specify this flag is for testing purposes only

Reply:
Done